src/data/finsupp/basic.lean

```diff
@@ -1000,6 +1008,12 @@ begin
   { rw [map_domain_notin_range, emb_domain_notin_range]; assumption }
 end
 
+@[to_additive]
+lemma prod_map_domain_index_inj [comm_monoid γ] {f : α → α₂} {s : α →₀ β}
```

Comment:
We don't abbreviate `inj` to `injective` if `function.injective` is used. See https://leanprover-community.github.io/contribute/naming.html#injectivity
```suggestion
lemma prod_map_domain_injective [comm_monoid γ] {f : α → α₂} {s : α →₀ β}
```

Reply:
Note that my naming here was intended to match `prod_map_domain_index` (above), and not match `map_domain_injective` (below). Should I remove the `_index` from the first lemma?

Reply:
That page says

> Injectivity lemmas written in terms of an injective **conclusion** should instead use the full word injective

This is an injective hypothesis. Should the page be clarified to include that?

Reply:
Oh, good point. I think the original name was fine then.